certbot-apache/certbot_apache/tests/configurator_test.py

```diff
@@ -812,31 +812,24 @@ def test_add_name_vhost_if_necessary(self):
         self.assertEqual(self.config.add_name_vhost.call_count, 2)
 
     @mock.patch("certbot_apache.configurator.http_01.ApacheHttp01.perform")
-    @mock.patch("certbot_apache.configurator.tls_sni_01.ApacheTlsSni01.perform")
     @mock.patch("certbot_apache.configurator.ApacheConfigurator.restart")
-    def test_perform(self, mock_restart, mock_tls_perform, mock_http_perform):
+    def test_perform(self, mock_restart, mock_http_perform):
```

Comment:
Can we simplify this test a bit more since the Apache plugin only supports 1 challenge type (and that's not likely to change anytime soon)?

Reply:
Done: